pkg/tests/apis/provisioning/testdata/timeline-demo.json

```diff
@@ -0,0 +1 @@
+../../../../../devenv/dev-dashboards/panel-timeline/timeline-demo.json
```

Comment:
If someone removes or changes that file, the resulting CI failure may not be easily correlated with the failing provisioning test. Should we use a file copy instead of a symlink, or create our own dashboard in that location?

Reply:
This is currently following suit with the other test files. For example: https://github.com/grafana/grafana/blob/67d2d2cb025cd2dc99f70e5ea84d44c2ec393a40/pkg/tests/apis/provisioning/testdata/all-panels.json

If we want to change this -- i think it would be good to change all of them at once than have a one-off here. wdyt?